autoform-inputs.js

```diff
@@ -325,8 +325,15 @@ updateAllTrackedFieldValues = function updateAllTrackedFieldValues(formId) {
 
 getAllFieldsInForm = function getAllFieldsInForm(template) {
   // Get all elements with `data-schema-key` attribute, unless disabled
-  return template.$("[data-schema-key]").not("[disabled]");
+  // return template.$("[data-schema-key]").not("[disabled]");
   // Exclude fields in sub-forms, since they will belong to a different AutoForm and schema.
   // TODO need some selector/filter that actually works correctly for excluding subforms
-  // return template.$('[data-schema-key]').not("[disabled]").not(template.$('form form [data-schema-key]'));
+  return template.$('[data-schema-key]').not("[disabled]").not(template.$('form form [data-schema-key]'));
```

Comment:
I think that's fine though if getAllFieldsInForm() should only select the fields for that form and not the subform? Then the subform will perform this same process and only select it's own direct inputs.